http.go

```diff
@@ -65,7 +65,7 @@ func HandleAuthenticated(router *httprouter.Router, method, path string, handle
 				http.Error(w, authMsg, authCode)
 				return
 			}
-			if !authToken.HasPermByID(permissions.PermEnabled) {
+			if authToken.IsInterService == nil || !(*authToken.IsInterService) || !authToken.HasPermByID(permissions.PermEnabled) {
```

Comment:
Under what conditions is `IsInterService == nil`? Because that condition causes the permission check to fail, even if it's a genuine user request.